### DISCUSSION TITLE
How to get the app free please ??

### DISCUSSION THREAD

**Discussion by JaiyeIam** [blinksh/blink#2025] [Support]
as a developper i really need this app in my daily life is it possible to get it free just using the terminal without build ????

**Comment by carloscabanero** [comment]
Hi! We do not offer any free plans. If you do not need Build, you still have access to the Blink+ yearly plan for $19.99/year, which is just $1.67/month. I think it is a fantastic deal if you are developer who needs the app.

If you want to be free and fully in control, the app is open source so you can compile it yourself as well.
